x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/esql/60_usage.yml

```diff
@@ -228,7 +228,7 @@ setup:
   - gt: {esql.functions.to_long: $functions_to_long}
   - match: {esql.functions.coalesce: $functions_coalesce}
   - gt: {esql.functions.categorize: $functions_categorize}
-  - length: {esql.functions: 146} # check the "sister" test above for a likely update to the same esql.functions length check
+  - length: {esql.functions: 137} # check the "sister" test above for a likely update to the same esql.functions length check
```

Comment:
checked that both these changes are good by running these tests in non-snapshot mode

```
 ./gradlew ":x-pack:plugin:yamlRestTest" --tests "org.elasticsearch.xpack.test.rest.XPackRestIT.test {p0=esql/60_usage/*}" -Dbuild.snapshot=false -Dtests.jvm.argline="-Dbuild.snapshot=false" -Dlicense.key=x-pack/license-tools/src/test/resources/public.key
```